website_snippet_contact_form/README.rst

```diff
@@ -8,6 +8,9 @@ Contact Form Snippet
 
 This module was written to extend the functionality of contact form to embed it
 in a snippet and allow you to put it in any page you want.
+You can choose if upload or not any file.
```

Comment:
"You can enable or disable uploading files."

Please add empty line to separate paragraphs.